tests/network/network_bonding_setup.pm

```diff
@@ -0,0 +1,116 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Setup simple network topology for bonding tests
+#
+# Maintainer: Volodymyr Katkalov <volodymyr.katkalov@suse.com>
+#
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use Utils::Systemd qw(disable_and_stop_service systemctl check_unit_file);
+use lockapi;
+use utils;
+
+my $BARRIER_MM_SETUP_DONE = 'MM_SETUP_DONE';
+my $BARRIER_BONDING_TESTS_DONE = 'BONDING_TESTS_DONE';
```

Comment:
I don't like this, IMO creating variable for name is just additional line of code.